transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionTransformers.java

```diff
@@ -55,6 +57,7 @@ public void registerTransformers(SubsystemTransformerRegistration subsystemRegis
         ResourceTransformationDescriptionBuilder builderEap7 = chainedBuilder.createBuilder(CURRENT_MODEL_VERSION, MODEL_VERSION_EAP70);
         builderEap7.getAttributeBuilder()
                 .setValueConverter(new AttributeConverter.DefaultValueAttributeConverter(OBJECT_STORE_RELATIVE_TO), OBJECT_STORE_RELATIVE_TO)
+                .setDiscard(DiscardAttributeChecker.DiscardAttributeValueChecker.ALWAYS, MAXIMUM_TIMEOUT)
```

Comment:
I would reject if it is defined and not equal to 300 (assuming 300 is what was used behind the scenes on the old servers). The way to do this is to use DiscardAttributeValueChecker to check/discard the value and then RejectAttributeChecker.DEFINED

Reply:
There is no MAXIMUM_TIMEOUT attribute on the old servers.

Reply:
@zhfeng sorry for the delay. In that case you should reject it if it is defined, and discard if undefined